buildkite-agent-hooks/remove-containers.sh

```diff
@@ -0,0 +1,14 @@
+echo "List of containers"
```

Comment:
For the sake of tools like [`shellcheck`](https://www.shellcheck.net), I'd suggest adding a `#!/bin/bash` to this and all other script files.